plugin/storage/kafka/options.go

```diff
@@ -74,15 +98,49 @@ func (opt *Options) AddFlags(flagSet *flag.FlagSet) {
 		defaultEncoding,
 		fmt.Sprintf(`Encoding of spans ("%s" or "%s") sent to kafka.`, EncodingJSON, EncodingProto),
 	)
+	flagSet.String(
+		configPrefix+suffixRequiredAcks,
+		defaultRequiredAcks,
+		"(experimental) Required kafka broker acknowledgement. i.e. noack, local, all",
+	)
+	flagSet.String(
+		configPrefix+suffixCompression,
+		defaultCompression,
+		"(experimental) Type of compression to use on messages. i.e none, gzip, snappy, lz4, zstd",
+	)
+	flagSet.Int(
+		configPrefix+suffixCompressionLevel,
+		defaultCompressionLevel,
+		"(experimental) compression level to use on messages. gzip = 1-9 (default = 6), snappy = none, lz4 = 1-17 (default = 9), zstd = -131072 - 22 (default = 3)",
```

Comment:
maybe just include the URL where these are defined. Or construct the description from the map[]struct defined above, so that they are always in sync.